src/lib/components/CarouselCard.svelte

```diff
@@ -0,0 +1,8 @@
+<!-- Carousel Card with filler content -->
+<div class="w-56 flex-initial rounded-full bg-blue p-5 text-center font-dm">
+    <!-- Sample Text -->
+    <p class="m-0">Lorem ipsum dolor sit amet consectetur</p>
+
+    <!-- Sample Logo -->
+    <!-- <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/Logo_of_YouTube_%282015-2017%29.svg/2560px-Logo_of_YouTube_%282015-2017%29.svg.png" alt="" class="m-auto w-40"> -->
```

Comment:
Normally, since we have version control, we can actually remove lines of code that are essentially commented out. There's no reason to keep them because that's what `git checkout` is for.
```suggestion
    <!-- Sample Logo -->
```